rakelib/artifacts.rake

```diff
@@ -61,11 +59,6 @@ namespace "artifact" do
       # See more in https://github.com/elastic/logstash/issues/4818
       "vendor/??*/**/.mvn/**/*",
 
-      # Without this when JRuby runs 'pleaserun' gem using the AdoptOpenJDK, during the post install script
-      # it claims that modules are not open for private introspection and suggest it's missing --add-opens
-      # so including these files JRuby run with modules opened to private introspection.
-      "vendor/jruby/bin/.jruby.java_opts",
```

Comment:
I think this is the cause of the :red_circle: CI:
- https://logstash-ci.elastic.co/job/elastic+logstash+pull-request+multijob-integration-2/4868/console
- https://logstash-ci.elastic.co/job/elastic+logstash+pull-request+multijob-integration-pq-2/4869/console

both fails with:
```
23:52:59     Failures:
23:52:59 
23:52:59       1) CLI > logstash-plugin prepare-offline-pack create a pack from a wildcard successfully create a pack
23:52:59          Failure/Error: expect(unpacked.plugins.collect(&:name)).to include(*filters)
23:52:59            expected ["logstash-filter-aggregate", "logstash-filter-anonymize", "logstash-filter-cidr", "logstash-filter-c...stash-filter-urldecode", "logstash-filter-useragent", "logstash-filter-uuid", "logstash-filter-xml"] to include "2022-03-10T22:30:59.176Z [main] WARN FilenoUtil : Native subprocess control requires open access to the JDK IO subsystem" and "Pass '--add-opens java.base/sun.nio.ch=org.jruby.dist --add-opens java.base/java.io=org.jruby.dist' to enable."
23:52:59          # ./specs/cli/prepare_offline_pack_spec.rb:92:in `block in <main>'
23:52:59          # /opt/logstash/build/qa/integration/vendor/jruby/2.5.0/gems/logstash-devutils-2.3.0-java/lib/logstash/devutils/rspec/spec_helper.rb:61:in `block in <main>'
23:52:59          # ./rspec.rb:37:in `<main>'
```

`vendor/jruby/bin/.jruby.module_opts` is needed by the JRuby run from `logstash-plugins`